011-php-code-formatting.md

```diff
@@ -0,0 +1,94 @@
+# PHP Code Formatting
+
+We should implement the use of both the [Prettier PHP Plugin](https://github.com/prettier/plugin-php) for prettifying the look of our PHP code and the [PHP Coding Standards Fixer](https://github.com/FriendsOfPHP/PHP-CS-Fixer) tool for ensuring the resulting PHP code is written well, following specified code quality rules.
+
+**Discussion:** <https://github.com/DoSomething/rfcs/pull/11>
+
+## Problem
+
+For most of our repositories that include PHP we currently depend on [StyleCI](https://styleci.io) for formatting our code.
+
+While it has served our needs for the most part, this tool is not as convenient as some other tools available, and at times has even been a bit confusing to use.
+
+The setup for StyleCI includes adding a GitHub repository to the list of projects it should "check" automatically for each "push" to a pull request branch. The settings can be customized on the StyleCI web interface but each project can also contain their own configuration in a `.styleci.yml` file. Having multiple locations for configuration settings can be a bit confusing; not always being apparent if a project is checked by StyleCI if the local config file is not present.
+
+When StyleCI runs to check the PHP code style on a push to a branch, if it finds errors, the run fails with a link to details in the pull request. At times these details are very confusing to understand exactly what the recommended change is. Developers sometimes need to try a few edits, commit them and see if the styles pass on subsequent runs, but there is no way to know without pushing up code.
+
+StyleCI does offer to create pull requests on demand for these changes, but that workflow feels a bit disjointed given that a developer would like this all fixed in their initial pull request.
+
+It would be ideal to be able to know exactly what needs to be changed and confident that doing so has fixed the style issue; and not needing to push changes numerous times to get feedback from StyleCI on whether the changes pass or not.
```

Comment:
This one is gonna be a game-changer. Those little :x:s always catch me at the last moment!!